e2e-tests/cypress/tests/integration/channels/menus/status_dropdown_spec.js

```diff
@@ -15,7 +15,7 @@ import theme from '../../../fixtures/theme.json';
 
 describe('Status dropdown menu', () => {
     const statusTestCases = [
-        {text: 'Online', className: 'icon-check', profileClassName: 'icon-check-circle'},
+        {text: 'Online', className: 'icon-check-circle', profileClassName: 'icon-check-circle'},
```

Comment:
@yasserfaraazkhan, This spec file has already been fixed and merged [PR-23258]( https://github.com/mattermost/mattermost-server/pull/23258)